test/functional/base58/__init__.py

```diff
@@ -0,0 +1,159 @@
+'''Base58 encoding
```

Comment:
To be clear, we never use base58... all our data encoding uses bech32.

Reply:
I see. It's only needed for `scalecodec` to work.